package.json

```diff
@@ -13,7 +13,6 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC",
```

Comment:
could also change the license "id" to "BSD-3-Clause"

Reply:
Done!